### DISCUSSION TITLE
Translations possible

### DISCUSSION THREAD

**Discussion by manuxi** [TheCadien/SuluNewsBundle#65] [General]
Are translations possible? I added the news bundle to a project but even in the preview the twig variable "localizations" is missing.

**Comment by TheCadien** [comment]
Unfortunately, this is not possible at the moment. 
However, I am currently building version 2.X of the bundle. In it I would like the BC very much that we enable the translations at least already for the content. 
Would you be interested to open a PR ?

**Comment by manuxi** [comment]
Well since this was urgent, I had to create a bundle by myself. Unfortunately, I also have no time at all right now ;)
